interchain/scripts/src/bin/create_remote_account.rs

```diff
@@ -0,0 +1,119 @@
+use abstract_client::AbstractClient;
+use abstract_core::objects::chain_name::ChainName;
+use abstract_core::objects::namespace::Namespace;
+use cw_orch::daemon::networks::neutron::NEUTRON_NETWORK;
+use cw_orch::daemon::networks::{ARCHWAY_1, JUNO_1, OSMOSIS_1, PHOENIX_1};
+use cw_orch::daemon::ChainKind;
+use cw_orch::prelude::*;
+use cw_orch::{
+    daemon::{ChainInfo, Daemon},
+    tokio::runtime::Handle,
+};
+use tokio::runtime::Runtime;
+
+/// <https://github.com/cosmos/chain-registry/blob/master/neutron/chain.json>
+pub const NEUTRON_1: ChainInfo = ChainInfo {
+    kind: ChainKind::Mainnet,
+    chain_id: "neutron-1",
+    gas_denom: "untrn",
+    gas_price: 0.075,
+    grpc_urls: &["http://grpc-kralum.neutron-1.neutron.org:80"],
+    network_info: NEUTRON_NETWORK,
+    lcd_url: Some("https://rest-kralum.neutron-1.neutron.org"),
+    fcd_url: None,
+};
+
+fn main() -> cw_orch::anyhow::Result<()> {
+    dotenv::dotenv()?;
+    env_logger::init();
+    let chains = vec![
+        (JUNO_1, None),
+        (PHOENIX_1, None),
+        (ARCHWAY_1, None),
+        (NEUTRON_1, None),
+        (OSMOSIS_1, Some(std::env::var("OSMOSIS_MNEMONIC")?)),
+    ];
+    let runtime = Runtime::new()?;
+
+    for src_chain in &chains {
+        for dst_chain in &chains {
+            if src_chain.0.chain_id != dst_chain.0.chain_id {
+                // Osmosis doesn't allow sending packets nowadays :/
+                if src_chain.0.chain_id != OSMOSIS_1.chain_id {
+                    // Neutron and Juno don't have an IBC-polytone connection :/
+                    if !(src_chain.0.chain_id == NEUTRON_1.chain_id
+                        && dst_chain.0.chain_id == JUNO_1.chain_id
+                        || src_chain.0.chain_id == JUNO_1.chain_id
+                            && dst_chain.0.chain_id == NEUTRON_1.chain_id)
+                    {
+                        connect(src_chain.clone(), dst_chain.clone(), runtime.handle())?;
+                    }
+                }
+            }
```

Comment:
1) Can we have a single `if` here instead?
2) For neutron-juno I would like to see a separate method which will have a list of unsupported(or supported) polytone connections and do something like match-loop instead

Reply:
I added a helper to know which chains are connected via Polytone. Is that better ?

Reply:
Yep